docs/docs/tutorials/noirjs_app.md

```diff
@@ -310,9 +327,7 @@ Time to celebrate, yes! But we shouldn't trust machines so blindly. Let's add th
 
 ```js
 display('logs', 'Verifying proof... ⌛');
-const verificationKey = await backend.getVerificationKey();
-const verifier = new Verifier();
-const isValid = await verifier.verifyProof(proof, verificationKey);
+const isValid = await backend.verifyProof(proof);
```

Comment:
Did we lose the ability to cache and verify with cached verification key?

Reply:
no we did not. I'll revert it back to how it was to make it clear that it is possible.

Reply:
added it as a comment